src/core/utils/compatibility.js

```diff
@@ -166,6 +166,13 @@ export function isCompatibleWithUserAgent({
     return { compatible: false, reason: INCOMPATIBLE_NOT_FIREFOX };
   }
 
+  // Fenix does not support add-ons (yet?).
```

Comment:
mmmmmmmmmm, that's not really true.

Reply:
Yeah, we should change it to say add-ons are not installable from AMO on Fenix yet.